pkg/telemetry/gather.go

```diff
@@ -233,3 +240,38 @@ func fetchKubernetesResources(ctx context.Context, cfg *config.Config) (map[stri
 	}
 	return metrics, nil
 }
+func getOSVersion() (string, error) {
+	file, err := os.Open("/etc/os-release")
+	if err != nil {
+		return "", fmt.Errorf("error opening /etc/os-release: %w", err)
+	}
+	defer file.Close()
+
+	var version string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "VERSION_ID=") {
+			version = strings.TrimPrefix(line, "VERSION_ID=")
+			version = strings.Trim(version, `"`)
+			break
+		}
+	}
+	if err := scanner.Err(); err != nil {
```

Comment:
What if by that point we already have `version`? I.e. error occurs after VERSION_ID=. Not sure if that even possible without some solar storms